### PR TITLE
zh-cn: web/performance: improve translation, fix format and links

### PR DESCRIPTION
**Pull request by pzhlkj6612**
### Description

I think it's better to keep the original English name of "transitions" and "animations" since they are proper nouns.

<br>

Then, here is one set of comparison:

Before:

<p><a href="/zh-CN/docs/Web/API/Window/requestAnimationFrame" title="requestAnimationFrame()"><code>requestAnimationFrame()</code></a>提供了一种用 JavaScript 代码制作动画的高效方式。本方法的回调函数在绘制下一帧之前由浏览器调用。与需要一个延迟参数的<a class="only-in-en-us" title="Currently only available in English (US)" href="/en-US/docs/Web/API/setTimeout"><code>setTimeout()</code> <small>(en-US)</small></a>/<a class="only-in-en-us" title="Currently only available in English (US)" href="/en-US/docs/Web/API/setInterval"><code>setInterval()</code> <small>(en-US)</small></a>相比，requestAnimationFrame() 效率高得多。开发人员可以在 requestAnimationFrame 回调函数中通过简单地改变元素的样式 (或者更新画布绘制，等等) 来创建动画。</p>

After:

<p><a href="/zh-CN/docs/Web/API/Window/requestAnimationFrame" title="requestAnimationFrame()"><code>requestAnimationFrame()</code></a> 提供了一种用 JavaScript 代码制作动画的高效方式。本方法的回调函数在绘制下一帧之前由浏览器调用。与需要一个延迟参数的 <a href="/zh-CN/docs/Web/API/setTimeout" title="setTimeout()" data-flaw-src="{{domxref(&quot;window.setTimeout&quot;,&quot;setTimeout()&quot;)}}"><code>setTimeout()</code></a> 或 <a href="/zh-CN/docs/Web/API/setInterval" title="setInterval()" data-flaw-src="{{domxref(&quot;window.setInterval&quot;,&quot;setInterval()&quot;)}}"><code>setInterval()</code></a> 相比，<code>requestAnimationFrame()</code> 效率高得多。开发人员可以在 <code>requestAnimationFrame()</code> 回调函数中通过简单地改变元素的样式 (或者更新画布绘制，等等) 来创建动画。</p>

### CODE REVIEW

File: files/zh-cn/web/performance/css_javascript_animation_performance/index.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
-title: CSS and JavaScript animation performance
+title: CSS 动画与 JavaScript 动画的性能
 slug: Web/Performance/CSS_JavaScript_animation_performance
 ---
 
-对众多应用程序而言，动画对提供友好的用户体验有着关键的作用，我们有很多方式生成 web 动画，比如 CSS {{cssxref("transition","transitions")}}/{{cssxref("animation","animations")}} 或者 JavaScript 动画 (使用 {{domxref("Window.requestAnimationFrame","requestAnimationFrame()")}}).
+对众多应用程序而言，动画对提供友好的用户体验有着关键的作用。我们有很多方式生成 web 动画，比如 CSS {{cssxref("transition","transitions")}} 和 {{cssxref("animation","animations")}} 或者基于 JavaScript 的动画 (使用 {{domxref("Window.requestAnimationFrame","requestAnimationFrame()")}})。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/performance/css_javascript_animation_performance/index.md:6]:
```suggestion
对众多应用程序而言，动画对提供友好的用户体验有着关键的作用。我们有很多方式生成 web 动画，比如 CSS {{cssxref("transition","transitions")}} 和 {{cssxref("animation","animations")}} 或者基于 JavaScript 的动画（使用 {{domxref("Window.requestAnimationFrame","requestAnimationFrame()")}}）。在这篇文章中，我们分析 CSS 动画和 JavaScript 动画的性能差异。
```

File: files/zh-cn/web/performance/css_javascript_animation_performance/index.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-在这篇文章中，我们分析 CSS 动画和 JavaScript 动画的性能差异
+在这篇文章中，我们分析 CSS 动画和 JavaScript 动画的性能差异。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/performance/css_javascript_animation_performance/index.md:8]: 最新的英文文档是合在一起的

File: files/zh-cn/web/performance/css_javascript_animation_performance/index.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 ## CSS 过渡和动画
 
-CSS 中过渡和动画都可以用于编写动画，它们都有各自的使用场景：
+CSS 中的 transitions 和 animations 都可以用于编写动画，它们都有各自的使用场景：
 
 - CSS {{cssxref("transition","transitions")}} 提供了一个简单的的方式去创造当前样式与结束状态样式之间的动画，比如一个 button 的普通状态和 hover 状态。尽管一个元素处于过渡状态中，新的过渡动画也会立即从当前样式开始，而不是直接跳转到 CSS 的最终状态。浏览 [Using CSS transitions](/zh-CN/docs/Web/Guide/CSS/Using_CSS_transitions) 以获取更多细节。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/performance/css_javascript_animation_performance/index.md:14]:
```suggestion
- CSS {{cssxref("transition","transitions")}} 提供了一个简单的的方式去创造当前样式与结束状态样式之间的动画，比如一个 button 的普通状态和 hover 状态。尽管一个元素处于过渡状态中，新的过渡动画也会立即从当前样式开始，而不是直接跳转到 CSS 的最终状态。浏览[使用 CSS transitions](/zh-CN/docs/Web/Guide/CSS/Using_CSS_transitions) 以获取更多细节。
```

File: files/zh-cn/web/performance/css_javascript_animation_performance/index.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 - 另一方面，CSS {{cssxref("animation","animations")}}, 允许开发者去通过一个初始状态属性值集合与最终状态属性值集合创造动画，而不是单单的初始和最终状态。CSS animations 由两部分组成：一个描述 CSS 动画的样式，以及一个指定动画初始状态，中间状态以及最终状态样式的关键帧集合。浏览 [Using CSS animations](/zh-CN/docs/Web/Guide/CSS/Using_CSS_animations) 以获取更多细节。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/performance/css_javascript_animation_performance/index.md:15]:
```suggestion
- 另一方面，CSS {{cssxref("animation","animations")}} 允许开发者去通过一个初始状态属性值集合与最终状态属性值集合创造动画，而不是单单的初始和最终状态。CSS animations 由两部分组成：描述 CSS 动画的样式，以及一组关键帧，表示动画样式的开始和结束状态，以及可能的中间状态。浏览[使用 CSS animations](/zh-CN/docs/Web/Guide/CSS/Using_CSS_animations) 以获取更多细节。
```

File: files/zh-cn/web/performance/css_javascript_animation_performance/index.md [hunk 5]
```diff
@@ -18,50 +18,50 @@ CSS 中过渡和动画都可以用于编写动画，它们都有各自的使用
...
 
 ## 性能对比：<br>transitions vs. requestAnimationFrame
 
-事实上，大多数场景下，基于 CSS 的 动画几乎是跟 JavaScript 动画表现一致 — 至少在 FireFox 上是如此。一些基于 Javascript 的动画库，像 [GSAP](http://greensock.com/gsap) and [Velocity.JS](http://velocityjs.org/), 甚至声称他们在性能上可以做得比 [native CSS transitions/animations](https://css-tricks.com/myth-busting-css-animations-vs-javascript/) （原生的 CSS transitions/animations）更好。这是可能的，因为在重绘事件发生之前，CSS transitions/animations 在主的 UI 线程仅仅是重新采集元素的样式，这跟通过 `requestAnimationFrame()` 回调，获取重新采集元素样式是一样的，也是在下一次重绘之前触发。假如二者都是在主 UI 线程创建的动画，那它们在性能方面没有差异。
+事实上，大多数场景下，基于 CSS 的 动画几乎是跟 JavaScript 动画表现一致 — 至少在 FireFox 上是如此。一些基于 Javascript 的动画库，像 [GSAP](http://greensock.com/gsap) 和 [Velocity.JS](http://velocityjs.org/), 甚至声称他们在性能上可以做得比 [native CSS transitions/animations](https://css-tricks.com/myth-busting-css-animations-vs-javascript/) （原生的 CSS transitions 和 animations）更好。这是可能的，因为在重绘事件发生之前，CSS transitions 和 animations 在主的 UI 线程仅仅是重新采集元素的样式，这跟通过 `requestAnimationFrame()` 回调，获取重新采集元素样式是一样的，也是在下一次重绘之前触发。假如二者都是在主 UI 线程创建的动画，那它们在性能方面没有差异。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/performance/css_javascript_animation_performance/index.md:29]:
```suggestion
事实上，大多数场景下，基于 CSS 的动画几乎是跟 JavaScript 动画表现一致——至少在 FireFox 上是如此。一些基于 Javascript 的动画库，像 [GSAP](http://greensock.com/gsap) 和 [Velocity.JS](http://velocityjs.org/)，甚至声称他们在性能上可以做得比[原生 CSS transitions/animations](https://css-tricks.com/myth-busting-css-animations-vs-javascript/)（原生的 CSS transitions 和 animations）更好。这是可能的，因为在重绘事件发生之前，CSS transitions 和 animations 在主的 UI 线程仅仅是重新采集元素的样式，这跟通过 `requestAnimationFrame()` 回调，获取重新采集元素样式是一样的，也是在下一次重绘之前触发。假如二者都是在主 UI 线程创建的动画，那它们在性能方面没有差异。
```

File: files/zh-cn/web/performance/css_javascript_animation_performance/index.md [hunk 6]
```diff
@@ -18,50 +18,50 @@ CSS 中过渡和动画都可以用于编写动画，它们都有各自的使用
...
 
-在这一节，我们将会使用 FireFox 通过一个性能测试，去看看哪种动画方式更好
+在这一节，我们将会使用 FireFox 通过一个性能测试，去看看哪种动画方式更好。
 
 ### 启用 FPS 工具
 
 在进行示例之前，请启用 FPS 工具先查看当前帧速率：
 
-1. 在地址栏中，输入 about:config;点击" I’ll be careful, I promise!"按钮，以输入配置屏幕。
+1. 在地址栏中，输入 *about:config*，点击“I'll be careful, I promise!”按钮，以输入配置屏幕。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/performance/css_javascript_animation_performance/index.md:37]:
```suggestion
1. 在地址栏中，输入 *about:config*，点击 `I'll be careful, I promise!` 按钮，以输入配置屏幕。
```

**Comment by pzhlkj6612** [comment]
Sorry for the late, I will finish this PR soon.

**Review by yin1999 — APPROVED** [review]
Thanks @pzhlkj6612, @Jack-Zhang-1314

I've applied some fixes to make this better

**Pull request merged**